assets/sass/helpers/_new_tab_links.scss

```diff
@@ -0,0 +1,27 @@
+// The following classes are used in assets/js/aplus.js. Therefore, you must
+// change the name of the classes in both files.
+
+.icon::after {
+	display: inline-block;
+	font-style: normal;
+	font-variant: normal;
+	text-rendering: auto;
+	-webkit-font-smoothing: antialiased;
+}
+
+.external-link::after {
+	font-family: "Font Awesome 5 Free";
+	font-weight: 900;
+	content: "\f35d";
```

Comment:
Is it necessary to use the `::after` pseudo-element? You already add new `<span>` elements in JS, so could the icon content be included directly in them? If that is infeasible, then of course the after pseudo-element is the right solution.

Reply:
The `<span>` that I added is `sr-only` and it only contains the text that is going to be read by the screen reader. I decided to use the pseudo-element  `::after`, because that symbol is a decorative feature. Of course, we can also add a second `<span>` or `<i>` element to add the icon using JS. Which one do you prefer?

Reply:
`::after` is OK. Let's keep it.